jbt-it/neomdb
# #312 client workshop instance details

How to test:

- navigate to a workshop instance detail page (e.g. Projektmanagement)
- check the "bearbeiten" button
- check the "anmelden" button
- check the add member feature
- check an instance with status "Feedback" or "Abgeschlossen" and check if the feedback button links to the feedback page (not yet available)
- check an instance with status "Anmeldung abgeschlossen" and check the button "Anwesenheitsliste eingeben" -> should show a table of all signed up members
- check an instance with status "Anmeldung" and check the button "Anmeldung schließen" -> should show a table of all signed up members
- check the participants and the function to add or remove a participant
- check the mobile version
